config/userRoutes.js

```diff
@@ -0,0 +1,91 @@
+const dbUser = require('../helpers/users-model');
+
+
+
+// users end points
+module.exports = server => {
+  server.get('/user', getAllUsers);
+  server.get('/user/:id', getUser);
+  server.post('/user', addUser);
+  server.delete('/user/:id', deleteUser);
+  server.put('/user/:id', updateUser);
+  server.get('/user/:id/trips', getTripsOfUser);  
+};
+
+// User routes
+const getAllUsers = (req, res) => {
+  dbUser.get()
+    .then(data => {
+      res.status(200).json(data);
+    })
+    .catch(err => {
+      res.status(500).json({ message: ` Failed to get all Users`, error: err });
+    });
+}
+
+const getUser = (req, res) => {
```

Comment:
You could stick these handler functions in another file (or multiple other files) for the sake of organization. You'll also see this be more common when working with other developers.